spdx/parsers/jsonyamlxml.py

```diff
@@ -1181,11 +1185,31 @@ def parse_pkg_down_location(self, pkg_down_location):
         else:
             self.value_error("PKG_DOWN_LOC", pkg_down_location)
 
+    def parse_pkg_files_analyzed(self, pkg_files_analyzed):
+        """
+        Parse Package files analyzed
+        - pkg_files_analyzed: Python boolean or str/unicode
+        """
+        # Files Analyzed optional
+        if pkg_files_analyzed is None:
+            return
+        if isinstance(pkg_files_analyzed, six.string_types) or isinstance(pkg_files_analyzed, bool):
+            try:
+                return self.builder.set_pkg_files_analyzed(
+                    self.document, six.text_type(pkg_files_analyzed)
+                )
+            except CardinalityError:
+                self.more_than_one_error("PKG_FILES_ANALYZED")
+        else:
+            self.value_error("PKG_FILES_ANALYZED", pkg_files_analyzed)
+
     def parse_pkg_verif_code_field(self, pkg_verif_code_field):
         """
         Parse Package verification code dict
         - pkg_verif_code_field: Python dict('value':str/unicode, 'excludedFilesNames':list)
         """
+        if self.document.packages[-1].files_analyzed == False:
```

Comment:
```suggestion
        if not self.document.packages[-1].files_analyzed:
```

Reply:
why testing on packages[-1]? Sounds like a hack assuming the current package being parser is the last one.

I think we may rather want to do this test in the caller function